_includes/icon.html

```diff
@@ -1,11 +1,14 @@
 {% comment %}
 include
 - style
+- icon_name
 - content
 {% endcomment %}
 <div class="usa-icon-list__icon {{include.style}}">
-  <svg role="img" class="usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"/></svg>
+  <svg role="img" class="usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Technically not a regression since they didn't exist before either, but I noticed these icons are lacking a text alternative and failing automated accessibility checks (using Axe extension). Maybe for a follow-up pull request if we want to keep the changes well-contained.

![image](https://github.com/18F/identity-dev-docs/assets/1779930/23803edd-67c7-41cd-9018-8bef3b72d1ab)


https://dequeuniversity.com/rules/axe/3.5/svg-img-alt

Reply:
since the icons in these lists are not semantically necessary (they are all followed by "DO" or "DON'T") i set the icon partial to use `aria-hidden=true`. that should i think be okay?

The checkmarks on the attributes page are the only ones that have semantic meaning, and since it's just the one icon, i didn't use the generic icon. i figured if the need for more semantic icons arises, we can make a different partial for that? but open to fix this in other ways if you have a preference!